File: WebAPI/IAI.Models/Models/Interviewer/Requests/InterviewerProfileModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IAI.Models.Models.Interviewer.Requests
{
    public class InterviewerProfileModel
    {
        public Guid InterviewerId { get; set; }
        public Guid InterviewerProfileId { get; set; }
        public string? ProfileName { get; set; }
        public int PrimarySkillId { get; set; }
        public string? PrimarySkillName { get; set; }
        public string? AdditionalSkills { get; set; }
        public List<InterviewerSkillsModel>? InterviewerSkills { get; set; }
    }

    public class InterviewerSkillsModel
    {
        public int SecondarySkillId { get; set; }
        public string? SecondarySkillName { get; set; }
        public int SecondarySkillNumber { get; set; }
    }
}

File: WebApp/iai-app/src/models/interviewer/request/InterviewerProfileModel.ts
interface InterviewerProfileModel {
    interviewerId: string;
    interviewerProfileId: string;
    profileName: string;
    primarySkillId: number;
    primarySkillName: string
    additionalSkills: string;
    interviewerSkills: InterviewerSkillsModel[];
}
export default InterviewerProfileModel;

interface InterviewerSkillsModel {
    secondarySkillId: number;
    secondarySkillName: string;
    secondarySkillNumber: number;
}